src/bfx_tools/topiary.ml

```diff
@@ -0,0 +1,198 @@
+open Biokepi_run_environment
+open Common
+
+type output_type = [
+  | `HTML of string
+  | `CSV of string
+]
+
+type predictor_type = [
+  | `NetMHC
```

Comment:
I don't know about mhctools but the wrapping itself can then depend on versions so it's fine (as long as software is clearly versioned :))